### PR TITLE
Updates to repo

### PR DESCRIPTION
**Pull request by ioreka**
WIP - updates to repo as outlined with @juliavallina

### CODE REVIEW

File: src/app/page.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    // Initializes data with an array containing an object with default values for the top orgs creating signatures
+    const [ranking_of_top_organizations_creating_signatures, setData] = useState([{
+        "organization": "",
+        "org_total": 0,
+    }]);
+   
+    // Initializes latency with an integer 0  
+    const [latency, setLatency] = useState(0);
+
+    // Define hardcoded date range for the query
```
Comment (ioreka) [inline src/app/page.js]: @juliavallina here's the section we worked on

File: src/app/components/activitiesByStatus.js
Comment (ioreka) [inline src/app/components/activitiesByStatus.js:1]: We provided these files, and I didn't create or need them as part of the tutorial. I assume they were provided as a "now you've made one component and completed the tutorial, here are some more for extra credit!" so obv very happy to reinstate them, just that this was what my repo looked like as a first time tutorial-follower

File: next.config.cjs
Comment (ioreka) [inline next.config.cjs:1]: we tell people to convert these from .js to .cjs, but it needs to be .mjs anyway which next.js now provides OOTB

File: src/app/page.js (hunk 1 above)
Comment (juliavallina) [reply src/app/page.js]: Ready!

File: src/app/components/activitiesByStatus.js
Comment (ioreka) [reply src/app/components/activitiesByStatus.js:1]: The components we are removing: 
image
I'm ok to remove them to move quickly, we can always reinstate them in future

**Pull request merged**